kettle-columnstore-bulk-exporter-plugin/src/main/java/com/mariadb/columnstore/api/kettle/KettleColumnStoreBulkExporterStep.java

```diff
@@ -4,7 +4,7 @@
  * Use of this software is governed by the Business Source License included
  * in the LICENSE file and at www.mariadb.com/bsl11.
  *
- * Change Date: 2021-04-01
+ * Change Date: 2022-01-30
```

Comment:
The change date can't be changed for a bug fix or minor release. Its set when we do the first major release.

Reply:
ok, set back to original value